admin-dev/themes/new-theme/js/notifications.js

```diff
@@ -63,26 +65,31 @@ const refreshNotifications = function () {
 let fillTpl = function (results, eltAppendTo, tpl) {
   eltAppendTo.children('.notification-elements').empty();
   if (results.length > 0) {
+    let router = new Router();
     eltAppendTo.removeClass('empty');
     $.each(results, function (property, value) {
       if (undefined === tpl) {
         return;
       }
 
-      eltAppendTo.children('.notification-elements').append(
-        tpl.replace(/_id_order_/g, parseInt(value.id_order))
-          .replace(/_customer_name_/g, value.customer_name)
-          .replace(/_iso_code_/g, value.iso_code)
-          .replace(/_carrier_/g, (value.carrier !== "" ? ` - ${value.carrier}` : ""))
-          .replace(/_total_paid_/g, value.total_paid)
-          .replace(/_id_customer_/g, parseInt(value.id_customer))
-          .replace(/_company_/g, (value.company !== "" ? ` (${value.company}) ` : ""))
-          .replace(/_date_add_/g, value.date_add)
-          .replace(/_status_/g, value.status)
-          .replace(/order_url/g, `${baseAdminDir}index.php?tab=AdminOrders&token=${token_admin_orders}&vieworder&id_order=${value.id_order}`)
-          .replace(/customer_url/g, `${baseAdminDir}index.php?tab=AdminCustomers&token=${token_admin_customers}&viewcustomer&id_customer=${value.id_customer}`)
-          .replace(/message_url/g, `${baseAdminDir}index.php?tab=AdminCustomerThreads&token=${token_admin_customer_threads}&viewcustomer_thread&id_customer_thread=${value.id_customer_thread}`)
-      );
+      var tplReplaced = tpl.replace(/_id_order_/g, parseInt(value.id_order))
+        .replace(/_customer_name_/g, value.customer_name)
+        .replace(/_iso_code_/g, value.iso_code)
+        .replace(/_carrier_/g, (value.carrier !== "" ? ` - ${value.carrier}` : ""))
+        .replace(/_total_paid_/g, value.total_paid)
+        .replace(/_id_customer_/g, parseInt(value.id_customer))
+        .replace(/_company_/g, (value.company !== "" ? ` (${value.company}) ` : ""))
+        .replace(/_date_add_/g, value.date_add)
+        .replace(/_status_/g, value.status)
+        .replace(/order_url/g, `${baseAdminDir}index.php?tab=AdminOrders&token=${token_admin_orders}&vieworder&id_order=${value.id_order}`)
+        .replace(/customer_url/g, `${baseAdminDir}index.php?tab=AdminCustomers&token=${token_admin_customers}&viewcustomer&id_customer=${value.id_customer}`);
```

Comment:
Nice catch @tdavidsonas88 ! I see there is also a link using legacy customer route although Customers page has been migrated as well. Could you please do the same fix for line 85 ? 😉

Reply:
Thank you @matks !  committed the change for legacy customer route and created function replaceGlobally = function (original, searchTxt, replaceTxt) for better better code reuse.